bazel/repository_locations.bzl

```diff
@@ -704,6 +704,19 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         extensions = ["envoy.wasm.runtime.wavm"],
         cpe = "cpe:2.3:a:webassembly_virtual_machine_project:webassembly_virtual_machine:*",
     ),
+    com_github_wamr = dict(
+        project_name = "Webassembly Micro Runtime",
+        project_desc = "A standalone runtime with a small footprint for WebAssembly",
+        project_url = "https://github.com/bytecodealliance/wasm-micro-runtime",
+        version = "0.1-beta",
+        sha256 = "e364c84654121e3b303d380e017540bb1b330845327db3f4d51fa353492ec04f",
+        strip_prefix = "wasm-micro-runtime-{version}",
+        urls = ["https://github.com/lum1n0us/wasm-micro-runtime/archive/refs/tags/v{version}.tar.gz"],
+        release_date = "2020-04-11",
+        use_category = ["dataplane_ext"],
+        extensions = ["envoy.wasm.runtime.wamr"],
+        cpe = "N/A",
+    ),
```

Comment:
New dependency additions need to adhere too: https://github.com/envoyproxy/envoy/blob/main/DEPENDENCY_POLICY.md /cc @envoyproxy/dependency-shepherds 

```
scorecard --repo=https://github.com/bytecodealliance/wasm-micro-runtime
                                                                                                                                          
RESULTS                                                                                                                                   
-------                                                                                                                                   
Repo: github.com/bytecodealliance/wasm-micro-runtime                                                                                      
Active: Pass 10                                                                                                                           
Branch-Protection: Fail 0                                                                                                                 
CI-Tests: Pass 10                                                                                                                         
CII-Best-Practices: Fail 10                                                                                                               
Code-Review: Pass 9                                                                                                                       
Contributors: Pass 10                                                                                                                     
Frozen-Deps: Fail 5                                                                                                                       
Fuzzing: Fail 10                                                                                                                          
Packaging: Fail 10                                                                                                                        
Pull-Requests: Pass 10                                                                                                                    
SAST: Fail 10                                                                                                                             
Security-Policy: Pass 10                                                                                                                  
Signed-Releases: Fail 0                                                                                                                   
Signed-Tags: Fail 10                                                                                                                      
```

Reply:
Fix to WAMR official Repo